tpl/tplimpl/embedded/templates/shortcodes/vimeo.html

```diff
@@ -4,11 +4,11 @@
 {{ template "_internal/shortcodes/vimeo_simple.html" . }}
 {{- else -}}
 {{ if .IsNamedParams }}<div {{ if .Get "class" }}class="{{ .Get "class" }}"{{ else }}style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;"{{ end }}>
-  <iframe src="https://player.vimeo.com/video/{{ .Get "id" }}" {{ if not (.Get "class") }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }}webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+  <iframe src="https://player.vimeo.com/video/{{ .Get "id" }}" {{ if not (.Get "class") }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }} {{ if .Get "title"}} title="{{ .Get "title" }}" {{ else }} title="vimeo video" {{ end }}  webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

Comment:
This is a little matter of taste, but it gets shorter an easier to maintain if you write something ala:

```
{{ with .Get "title" }} {{ . }}{{ else }}{{ end }}
```

Other than that it looks good. But you need to run `mage generate` to regenerate the Go code. This will also tell us if there are test failures.